flyteadmin/pkg/manager/mocks/project.go

```diff
@@ -6,12 +6,14 @@ import (
 	"github.com/flyteorg/flyte/flyteidl/gen/pb-go/flyteidl/admin"
 )
 
+type GetDomainsFunc func(ctx context.Context, request admin.GetDomainRequest) *admin.Domains
```

Comment:
ditto, could we move it below to GetProject?

Reply:
resolved